### ISSUE TITLE
Minimize jQuery use in javascript files

### ISSUE THREAD

**Issue by mworrell** [zotonic/zotonic#1558]
See also #908

A note from @ArthurClemens:

 * From (most) jQuery to vanilla: http://youmightnotneedjquery.com
 * Selectors: http://blog.garstasio.com/you-dont-need-jquery/selectors/
 * JavaScript documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript - also lists browser support and polyfills for older browsers.

**Comment by ulrikaugustsson** [comment]
Is this something you want outside help with?

**Comment by ddeboer** [comment]
Hey @ulrikaugustsson, that would be most welcome! Do you need some pointers where you could start?

**Comment by ulrikaugustsson** [comment]
@ddeboer yes, that would be great! I'm not *that* in to Erlang yet so if there is anything that is coupled to Erlang I might need some further help.

I can hopefully get to this during the week.
